lib/improver/tests/spotdata/spotdata/test_extract_data.py

```diff
@@ -46,11 +46,14 @@
 from iris.cube import Cube
```

Comment:
Because spotdata in the lib is in /lib/spotdata, so it ends up two deep in tests, c.f. /lib/nbhood --> /tests/nbhood/nbhood etc.